docs/index.md

```diff
@@ -36,7 +41,28 @@ ONNX models and functions:
 Note that the runtime is intended to help understand and debug function definitions.
 Performance is not a goal here.
 
-### Example
+## Installing ONNX Script
+
+```bash
+pip install --upgrade onnxscript
+```
+
+### Install for Development
+
+```bash
+pip install onnx onnxruntime pytest
+git clone https://github.com/microsoft/onnxscript
+cd onnxscript
+pip install -e .
+```
+
+### Run Unit Tests
+
+```bash
+pytest onnxscript
+```
```

Comment:
I think we can simply point developers to the repo readme, since the doc is more user facing